src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -49,6 +49,7 @@ public WorkspaceInfo()
                 double cx = 0;
                 double cy = 0;
                 double zoom = 1.0;
+                double scaleFactor = 1.0;
```

Comment:
The scale factor is passed into LibG using `WorkspaceEvents` in `DynamoServices`. These are the related PR's to enable this:
https://github.com/DynamoDS/Dynamo/pull/6967
https://github.com/DynamoDS/Dynamo/pull/7391